polyply/data/martini2/citations.bib

```diff
@@ -70,3 +70,10 @@ @article{P3HT_M2
   doi={10.1021/jacs.6b11717}
 }
 
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```

Comment:
following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```